sql/src/main/java/io/crate/action/sql/RowReceiverToResultReceiver.java

```diff
@@ -27,7 +27,7 @@
 
 import java.util.Set;
 
-class RowReceiverToResultReceiver implements RowReceiver {
+public class RowReceiverToResultReceiver implements RowReceiver {
```

Comment:
Hm too bad. Guess it's okay for now then, but I think we should re-consider the simpleQuery implementation and maybe get back to calling into the other session methods.